data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/AuthConfig.java

```diff
@@ -54,25 +54,29 @@ public boolean hasOnlyOneConfig() {
     }
 
 
-    public static class SslAuthConfig {
-        // TODO Add Support for SSL authentication types like
-        // one-way or two-way authentication
+    /*
```

Comment:
Take it or leave it: I would recommend deleting this dead code and leave a single like comment instead of commenting it out a large code block with no description for the TODO.